### PR TITLE
Feat Add App CPU level metrics

### PR DESCRIPTION
**Pull request by Mubangizi**
#### what this PR does
- Adds CPU level metrics for apps
- Adds them to route `/projects/<project_id>/apps/<app_id>/metrics/cpu`
- Adds error handling for the previous metrics for projects for empty values case

### CODE REVIEW

File: app/controllers/app.py [hunk 1]
```diff
@@ -1065,3 +1091,70 @@ def delete(self, app_id):
...
+            step=step,
+            metric='sum(rate(container_cpu_usage_seconds_total{container!="POD", image!="", namespace="' +
+            namespace+'", pod=~"'+app_alias+'.*"}[5m]))'
+        )
+        #  change array values to json"values"
+        new_data = json.loads(prom_data)
+        cpu_data_list = []
+        try:
+            for value in new_data["data"]["result"][0]["values"]:
+                case = {'timestamp': value[0], 'value': value[1]}
```
Comment (MutegekiHenry) [inline app/controllers/app.py:1155]: don't we have to convert these values to floats for ease of use by frontend

Comment (Mubangizi) [reply app/controllers/app.py]: I have applied the changes

**Comment by inno-asiimwe** [comment]
@Mubangizi , this PR has so many changes not related to your work. Mostly linting changes. Please revert those. Only changes related to your future should be included.

**Comment by Mubangizi** [comment]
EEh @inno-asiimwe  how do you suppose I do that?? I go line by line removing spaces??

**Comment by Mubangizi** [comment]
I have fixed the merge requests

**Pull request merged**